.pipelines/pull-request.yml

```diff
@@ -10,16 +10,21 @@ pr:
   drafts: true
 
 pool:
-  vmImage: windows-latest
+  vmImage: vs2017-win2016
```

Comment:
Just wanted to add that we can consider trying `windows-latest` once the `sass-extract` dependency supports `dart-sass`, which should remove the dependency of `node-sass`/`node-gyp 3.8` https://github.com/jgranstrom/sass-extract/issues/43